changelog/fix-woopay-tracker-undefined-wc-store-id

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fixes incorrect defined statement for WC_Install::STORE_ID_OPTION constant.
```

Comment:
Nitpick but we could avoid the changelog since it's fixing things within the release. The current approach isn't critical, though.